src/main/java/hudson/plugins/sidebar_link/ProjectLinks.java

```diff
@@ -55,6 +55,12 @@ public ProjectLinks(List<LinkAction> links) {
 
     public List<LinkAction> getLinks() { return links; }
 
+    public Collection<? extends Action> getJobActions(AbstractProject<?,?> job) {
+        if(links == null)
+            return new ArrayList<LinkAction>();
```

Comment:
Nit picking, but better to return Collections.emptySet() so you're not doing unnecessary instantiation